packages/addons-manager/bundle/config/upstream/addons-manager.yaml

```diff
@@ -359,8 +359,8 @@ spec:
         name: tanzu-addons-controller
         resources:
           limits:
-            cpu: 100m
-            memory: 500Mi
+            cpu: 500m
+            memory: 1200Mi
```

Comment:
Maybe add a comment on the commit how the numbers were picked. it already mentions  a bit of why, but fo rgure reference would be nice to document how  and why to change it if it is possible that in the future it needs to be changed again.